website/src/app/pricing/page.tsx

```diff
@@ -0,0 +1,726 @@
+                    the same network connectivity context. Typically a subnet or
+                    VPC.
+                    <div className="tooltip-arrow" data-popper-arrow></div>
+                  </div>
+                </td>
+                <td className="px-6 py-4">3</td>
+                <td className="px-6 py-4">No limit</td>
+              </tr>
+              <tr className="border-b border-1 border-neutral-200">
+                <td className="px-6 py-4">
+                  <span
+                    data-tooltip-target="policies-tooltip"
+                    data-tooltip-placement="top"
+                    className="underline hover:no-underline cursor-help"
+                  >
+                    Policies
+                  </span>
+                  <div
+                    id="policies-tooltip"
+                    role="tooltip"
+                    className="absolute z-10 invisible inline-block px-3 py-2 text-xs font-medium text-white transition-opacity duration-100 bg-neutral-900 rounded shadow-sm opacity-90 tooltip"
+                  >
+                    Policies control access to resources (e.g. group “A” may
+                    access resource “B”)
+                    <div className="tooltip-arrow" data-popper-arrow></div>
+                  </div>
+                </td>
+                <td className="px-6 py-4">No limit</td>
+                <td className="px-6 py-4">No limit</td>
+              </tr>
+              <tr className="border-b border-1 border-neutral-200">
+                <td className="px-6 py-4">
+                  <span
+                    data-tooltip-target="resources-tooltip"
+                    data-tooltip-placement="top"
+                    className="underline hover:no-underline cursor-help"
+                  >
+                    Resources
+                  </span>
+                  <div
+                    id="resources-tooltip"
+                    role="tooltip"
+                    className="absolute z-10 invisible inline-block px-3 py-2 text-xs font-medium text-white transition-opacity duration-100 bg-neutral-900 rounded shadow-sm opacity-90 tooltip"
+                  >
+                    Anything you wish to manage access to (e.g. database, VPC,
+                    home network, web server, SaaS application)
+                    <div className="tooltip-arrow" data-popper-arrow></div>
+                  </div>
+                </td>
+                <td className="px-6 py-4">No limit</td>
+                <td className="px-6 py-4">No limit</td>
+              </tr>
+              <tr className="border-b border-1 border-neutral-200">
+                <td className="px-6 py-4">
+                  <span
+                    data-tooltip-target="devices-tooltip"
+                    data-tooltip-placement="top"
+                    className="underline hover:no-underline cursor-help"
+                  >
+                    Devices
```

Comment:
In the app we're calling these 'Clients' is there any concern about confusion?

Reply:
Good catch, yeah it crossed my mind. I used `Devices` here because that's more common parlance with other products in the space, but I think it's better to be consistent with our terminology. Updated this to `Connected Clients` and the helptext to match.